src/styles/theme/default-theme.css

```diff
@@ -0,0 +1,33 @@
+/*
+We use the `postcss-functions` plugin to allow us to write functions in JS and then execute them in our CSS files.
+*/
+:root {
+  --lc-color-transparent: transparent;
+  --lc-color-current: currentColor;
+  --lc-color-black: hexToRGB(#000);
```

Comment:
@jaredhill4 you can use a javascript function here because it's postCSS? but in other projects, we'll need to document that these theme files should only utilize RGB colors right?

Reply:
@jaruesink, yes, that's correct. And they would be in the following format, since the Tailwind config wraps them in the `rgba()` function:

```css
--lc-color-black: 0, 0, 0;
```